docs/README-develop-best-practices.md

```diff
@@ -239,7 +253,7 @@ Having this logic defined in the common object lets the minitest easily check th
 
 Without this equality operator `==` only passes if they are the same instance object.  With this equality operator `==` passes if they are different objects referring to the same configuration on the node.
```

Comment:
"The built-in equality operator `==` returns true only if they are the same instance object.  The `==`method below is used to override the built-in equality operator, which returns true even when they are different objects referring to the same configuration on the node."